website/docs/learn-graphql/guides/the-design-of-graphql.md

```diff
@@ -41,7 +33,7 @@ The service sends some data down to the client, `ChatList` passes it on to its c
 
```

Comment:
It seems like you're already assuming at this point that the top level component is the only one making a request to the backend service, since you talk about `ChatList` being the conduit for the data to get to its child components. Theoretically they could make their own queries (and I see this a lot in real code). It might be worth explaining this assumption or even working through why that is a bad pattern first.

Reply:
With the order of the guides having changed, let's see how this reads.